src/components/common/StaticHomePage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Awesome implementation of static home page content for User. May add more sections in future, but this is a great start. Keep notes on possible sections to add later on.